homeassistant/components/python_script.py

```diff
@@ -11,7 +11,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({})
+    DOMAIN: vol.Schema(dict)
```

Comment:
Actually, I did want to validate against an empty dictionary. There is no config available.